packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.tsx

```diff
@@ -1299,10 +1370,17 @@ export class DatePicker {
     return inputProps;
   };
 
-  render() {
+  renderHiddenInput = () =>
+    !this.showDateInput ? (
+      <span id={this.selectedDateInfoId} class="sr-only" aria-live="polite">
+        <span ref={(el) => (this.selectedDateInfoEl = el)} role="status"></span>
+      </span>
+    ) : (
+      ""
+    );
+
+  renderCalendar = () => {
```

Comment:
```suggestion
  private renderCalendar = () => {
```
needed to resolve stenciljs lint error

Reply:
Fixed.